Source/BugsnagCrashReport.m

```diff
@@ -452,6 +471,20 @@ - (NSDictionary *)serializableValueWithTopLevelData:
   BSGDictSetSafeObject(event, [self app], @"app");
   BSGDictSetSafeObject(event, [self context], @"context");
   BSGDictInsertIfNotNil(event, self.groupingHash, @"groupingHash");
+    
+    BSGDictSetSafeObject(event, @(self.handledState.unhandled), @"unhandled");
+    
+    // serialize handled/unhandled into payload
+    NSMutableDictionary *severityReason = [NSMutableDictionary new];
+    NSString *reasonType = [BugsnagHandledState stringFromSeverityReason:self.handledState.calculateSeverityReasonType];
+    severityReason[@"type"] = reasonType;
+    
+    if (self.handledState.attrKey) {
```

Comment:
It looks like `attrValue` should be checked too, since a dict value can't be nil